framework/generated/khronos_generators/vulkan_generators/encode_pnext_struct_generator.py

```diff
@@ -114,11 +114,11 @@ def beginFile(self, gen_opts):
             file=self.outFile
         )
         write(
-            '    while ((base != nullptr) && ((base->sType == VK_STRUCTURE_TYPE_LOADER_INSTANCE_CREATE_INFO) ||',
+            '    while ((base != nullptr) && ((base->{} == VK_STRUCTURE_TYPE_LOADER_INSTANCE_CREATE_INFO) ||'.format(current_api_data.struct_type_variable),
```

Comment:
In a future commit, this is moved into a special handle case done by each API version of the script.  Since each API will have one or more of it's own structures to ignore.